cpp/include/cudf/table/table_view.hpp

```diff
@@ -336,6 +336,22 @@ inline bool has_nested_nulls(table_view const& input)
   });
 }
 
+/**
+ * @brief Returns True if the table is nullable in any of its columns hierarchy
+ *
+ * @param input The table to check for nullables
+ * @return True if the table has nullables in any of its columns hierarchy, false otherwise
+ */
+inline bool has_nested_nullables(table_view const& input)
```

Comment:
It's weird to name nullable columns "nullables" in my opinion. Can we rename to "nullable columns"?
```suggestion
/**
 * @brief Returns True if the table has a nullable column at any level of the column hierarchy
 *
 * @param input The table to check for nullable columns
 * @return True if the table has nullable columns at any level of the column hierarchy, false otherwise
 */
inline bool has_nested_nullable_columns(table_view const& input)
```